jsonmodels/__init__.py

```diff
@@ -1 +1,2 @@
 # coding: utf-8
+__version__ = "2.8.4"
```

Comment:
I couldn't find a better way to set the version, in the metadata it's still stuck at 2.4

Reply:
yeah unfortunately this is how Python/pip deals with package versions